### PR TITLE
Add required `device_type` to inventory data.

### PR DESCRIPTION
**Pull request by pasinskim**
Signed-off-by: Marcin Pasinski <marcin.pasinski@mender.io>

### CODE REVIEW

File: mender.go [hunk 1]
```diff
@@ -165,35 +165,41 @@ func NewMender(config menderConfig, pieces MenderPieces) (*mender, error) {
...
 	}
 	if err := scanner.Err(); err != nil {
 		log.Error(err)
 	}
-	return imageID
+	return ""
+}
+
+func (m mender) GetCurrentImageID() string {
+	return m.getManifestData("IMAGE_ID")
```
Comment (bboozzoo) [inline mender.go]: are we interested in reporting IMAGE_ID too?

File: inventory_data.go [hunk 2]
```diff
@@ -67,6 +67,20 @@ func listRunnable(dpath string) ([]string, error) {
 	return runnable, nil
 }
 
+func (id *InventoryDataRunner) AddDeviceType(device string, inv InventoryData) error {
```
Comment (bboozzoo) [inline inventory_data.go]: how about adding a separate method to InventoryData that adds custom keys, say:
```
func (i *InventoryData) AddAttributes(map[string]interface{}) {
    ...
}
```

File: mender.go (hunk 1 above)
Comment (pasinskim) [inline mender.go]: Since we are having it already maybe we can report it as well.

File: inventory_data.go (hunk 2 above)
Comment (pasinskim) [inline inventory_data.go]: This is a bit tricky. I wanted to add a device type only it this is not there already. Maybe instead we should replace existing one with the arbitrary one you are adding here.

**Pull request merged**